test/clojure_for_js_devs/handlers_test.clj

```diff
@@ -1,8 +1,14 @@
 (ns clojure-for-js-devs.handlers-test
   (:require [clojure.test :refer [deftest is testing]]
-            [clojure-for-js-devs.handlers :as handlers]))
+            [clojure-for-js-devs.handlers :as handlers]
+            [bond.james :as bond]
+            [clojure-for-js-devs.redis :as redis]))
 
 (deftest handlers
```

Comment:
Let's have different deftests for each handler - this is the pattern we follow in growth-auth-service